src/app/shared/components/template/components/tile-component/tile-component.component.scss

```diff
@@ -41,7 +41,7 @@ $background-secondary-light: var(
       --box-shadow: none;
 
       box-shadow: none;
-      --ripple-color: transparent;
+      // --ripple-color: transparent; // why was this there?
```

Comment:
I think it's there to avoid the unexpected behaviour of the icon having a ripple effect when you click it, but not the whole "tile" element. Since the component uses an `ion-button` within it, this sub-element gets the ripple effect applied automatically. I've updated the [example_tiles](https://docs.google.com/spreadsheets/d/1UWZDMskUnAxkGMmEv9uEpFetXQLdMRdu2ecXEKcbTjo/edit?gid=1595229908#gid=1595229908) template so that the image links are no longer broken. Uncomment this line and experiment with it yourself, it's strange that just the icon has an effect when clicked.

I'm not entirely sure why the component is written this way, using an `ion-button` as a wrapper around where it could just render the icon in a `div`. I suspect this is just some old code that could be rewritten. However, I don't think that's worth worrying about – I think you should revert the changes to the `tile-component` files for now so that this PR only affects the `button` component.

Also, for comments like this that are intended for the reviewer of the PR, they should be added as a comment within github once you've created the PR (I can show you how to do this), rather than in the code itself. If we were to merge these changes, this comment would be permanently committed to the codebase and wouldn't make sense to someone coming across it without any context.

Reply:
Oh whoops, I forgot to remove that comment. When I checked, I think it made the ripple not easy to see and once I removed it, it became visible and had the default colour it needed.